README.md

```diff
@@ -105,12 +128,27 @@ to discard information. The following filters are provided:
   enabling both the cgl and glx module on Mac OS X.
 * SessionFilter discards all resources not belonging to a given session
 
-## Projects using HW-SD
+# Projects using HW-SD {#Projects}
 
 * [Equalizer](http://www.equalizergraphics.com) parallel rendering
   framework. ([source](https://github.com/Eyescale/Equalizer/blob/master/libs/eq/server/config/resources.cpp#L61))
 
-## Further Links
+# Further Links {#Links}
 
 * [Unimplemented Features](https://github.com/Eyescale/hwsd/issues?labels=Feature)
 * [Known Bugs](https://github.com/Eyescale/hwsd/issues?labels=Bug)
+
+# Changes {#Changes}
+
+## Version 1.2 {#Changes12}
+
+* [38](https://github.com/Eyescale/hwsd/pull/38): OS X: only find main
+  display in hwsd_gpu_cgl module
+* [37](https://github.com/Eyescale/hwsd/pull/37): MSVC: Fix static init deadlock
+* [33](https://github.com/Eyescale/hwsd/pull/33): Port to Qt5
+
+## Version 1.1 {#Changes11}
```

Comment:
put date like in https://github.com/Eyescale/Collage/blob/master/doc/Changelog.md?